cleanlab_studio/studio/enrichment.py

```diff
@@ -0,0 +1,585 @@
+        latest_job = self._get_latest_job()
+        if latest_job["job_type"] != "ENRICHMENT":
+            # TODO: consider fetching latest populate job directly instead of throwing an error
+            # This would prevent the user from getting stuck in an error state if preview job is the latest job
+            raise ValueError(
+                "The latest job is a preview, to execute against entire dataset, please do `run()` first."
+            )
+        status = latest_job["status"]
+        if status == EnrichmentJobStatusEnum.FAILED.value:
+            raise ValueError("The latest populate job failed.")
+        elif status == EnrichmentJobStatusEnum.PAUSED.value:
+            raise ValueError(
+                "The latest populate job is paused, likely due to quota limit. Please contact us to discuss your use case - support@cleanlab.ai."
+            )
+        elif status in {
+            EnrichmentJobStatusEnum.RUNNING.value,
+            EnrichmentJobStatusEnum.CREATED.value,
+        }:
+            return False
+        elif status == EnrichmentJobStatusEnum.SUCCEEDED.value:
+            return True
+        else:
+            raise ValueError("The latest populate job has an unknown status.")
+
+    def wait_until_ready(self) -> None:
+        """Wait until the latest populate job is ready."""
+        latest_job_status = self._get_latest_job_status()
+        num_rows = latest_job_status["num_rows"]
+        spinner = itertools.cycle("|/-\\")
+        with tqdm(
+            total=num_rows,
+            desc="Enrichment Progress:",
+            bar_format="{desc} Rows Processed - {n_fmt}/{total_fmt}{postfix}",
+        ) as pbar:
+            while not self.ready:
+                latest_job_status = self._get_latest_job_status()
+                self._update_progress_bar(pbar, latest_job_status)
+                for _ in range(CHECK_READY_INTERVAL):
+                    time.sleep(0.1)
+                    pbar.set_description_str(f"Enrichment Progress: {next(spinner)}")
+
+                if latest_job_status.get("error"):
+                    raise EnrichmentProjectError(
+                        f"Project {self.id} failed to complete. Error: {latest_job_status['error']}"
+                    )
+            latest_job_status = self._get_latest_job_status()
+            self._update_progress_bar(pbar, latest_job_status)
+
+    def _update_progress_bar(self, pbar: tqdm, latest_job_status: dict[str, Any]) -> None:
+        num_processed_rows = latest_job_status["processed_rows"]
+        average_trustworthiness_score = latest_job_status["average_trustworthiness_score"]
+        pbar.set_postfix_str(
+            f"Average Trustworthiness Score: {average_trustworthiness_score}, Status: {latest_job_status['status']}"
+        )
+        pbar.update(num_processed_rows - pbar.n)
+
+    def download_results(
+        self, job_id: Optional[str] = None, include_original_dataset: Optional[bool] = False
```

Comment:
```suggestion
        self, 
        *,
        job_id: Optional[str] = None, 
        include_original_dataset: Optional[bool] = False
```

I would make it all required keywords here